pkg/arvo/sys/vane/eyre.hoon

```diff
@@ -707,44 +707,58 @@
     ::
         %scry
       (handle-scry authenticated address request(url suburl))
+    ::
+        %name
+      (handle-name authenticated request)
     ::
         %four-oh-four
       %^  return-static-data-on-duct  404  'text/html'
       (error-page 404 authenticated url.request ~)
     ==
+  ::  +handle-name: respond with our @p or 403
+  ::
+  ++  handle-name
+    |=  [authenticated=? =request:http]
+    ^-  (quip move server-state)
+    ?.  authenticated
+      (error-response authenticated url.request 403 ~)
+    ?.  =(%'GET' method.request)
+      (error-response authenticated url.request 405 "may only GET name")
+    %^  return-static-data-on-duct  200  'text/plain'
```

Comment:
Any thoughts on the mime type here? Debated between this and `application/json`, also aware of the upcoming scrying raw nouns stuff.

Reply:
`text/plain` is simplest, I think this is good.